testing/conftest.py

```diff
@@ -6,6 +6,8 @@
 import re
 import sys
 
+import pygments
```

Comment:
This makes the testsuite explicitly depend on pygments - but the tests in `test_terminal.py` already fail if `pygments` is not installed, and it's in `optional-dependencies.dev` in `pyproject.toml`, so I suppose that's alright?

Reply:
Seems fine to me.

The alternative would be to import-or-skip inside the fixture function.